_2019/backups.md

```diff
@@ -42,7 +42,7 @@ A proper backup system should be versioned in order to prevent this failure mode
 
 ## Deduplication
 
-However, making several copies of your data might be extremely costly in terms of disk space. Nevertheless, from one version to the next, most data will be identical and needs not be transferred again. This is where [data deduplication](https://en.wikipedia.org/wiki/Data_deduplication) comes into play, by keeping track of what has already been stored one can do **incremental backups** where only the changes from one version to the next need to be stored. This significantly reduces the amount of space needed for backups beyond the first copy.
+However, making several copies of your data might be extremely costly in terms of disk space. Nevertheless, from one version to the next, most data will be identical and needs not be transferred again. This is where [data deduplication](https://en.wikipedia.org/wiki/Data_deduplication) comes into play, by keeping track of what has already been stored one can do **incremental backups** where only the changes from one version to the next need to be stored. This significantly reduces the amount of space needed for backups beyond the first copy. MacOS Time Machine performs deduplicated backups.
```

Comment:
I don't think this addition is necessary :)

Reply:
That’s fair, I simply added it because I wasn’t sure whether or not Time Machine did deduplicated backups, and I found out by backing up my machine. Probably others might wonder the same thing.

Reply:
Yeah, it just seems a little out of place to mention one particular backup tool here.